### ISSUE TITLE
lint: commentedOutCode false negative on multi-statements

### ISSUE THREAD

**Issue by quasilyte** [go-critic/go-critic#668]
Snippet below has 3 commented-out code lines:
```go
	//rulebase resources
	ruh := rulebases.Handler{Db: datab, WebDir: conf.WebDir, Checker: checker}
	rulebases := api.Group("/rulebases", jwtMiddleware) //base URI
	rulebases.GET("", ruh.GetRulebases)                 //Returns a dictionary with all available Rulebases
	//rulebases.POST("/", postRsHandler)                                //create a rulebase
	//rulebases.DELETE("/:id", deleteRsHandler)                         //delete a rulebase
	//rulebases.PUT("/:setid", putRsHandler)                            //update a rulebase
	rulebases.PUT("/:baseid/documents", ruh.CheckDoc)               //process provided document against rulebase
	rulebases.PUT("/:baseid/documents/:documentid", ruh.CheckDocID) //process document against rulebase
```
They are not detected by the `commentedOutCode` for whatever reason.
They should be reported.

**Comment by Quasilyte** [comment]
This code also not detected:
```go
	/*
		resp, err := ioutil.ReadAll(c.Request().Body)
		if err != nil {
			e := err.Error()
			return c.JSON(http.StatusNotFound, structs.Response{Ok: false, Reason: &e})
		}
		fmt.Println(string(resp))
		return c.JSON(http.StatusNotFound, structs.Response{Ok: false})
	*/
```

**Comment by Quasilyte** [comment]
```go
/*
		ethrMsg := createBgnMsg(lport)
		sendSessionMsg(test.enc, ethrMsg)
		sendSessionMsg(test.enc, ethrMsg)
*/

// ui.printErr(err)
// return
```
